apps/registration-api/RegistrationApi.js

```diff
@@ -18,49 +18,46 @@ app.use(morgan('combined'))
 
 app.get('/register', regAuthCheck, (req, res) => {
 
+  const userName = req.get('User')
   const tmpObj = tmp.dirSync()
   const tmpDir = tmpObj.name
 
   exec(
     `/usr/bin/create-key-cert.sh -b beehive-dev -e +1d -o ${tmpDir} -c ${CA_KEY}`,
     (err) => {
       if (err) {
-        return res.status(500).send({ error: 'could not run script' })
+        let error = 'could not run script'
+        console.log(error)
```

Comment:
I would suggest logging the actual error so that you know exactly what went wrong when looking at the logs.